commercetools-models/src/main/java/io/sphere/sdk/products/ProductDraft.java

```diff
@@ -62,12 +69,13 @@
 
     /**
      * Flag for publishing the product immediately.
-     *
+     * <p>
      * {@include.example io.sphere.sdk.products.commands.ProductCreateCommandIntegrationTest#createPublishedProduct()}
      *
      * @return true if product should be published
      */
     @Nullable
+    @JsonProperty
```

Comment:
Just for consistency with our code and to make the name of this property more explicit, I would suggest to add the value "publish" to this annotation: `@JsonProperty("publish")`